mindsdb/integrations/handlers/cohere_handler/__init__.py

```diff
@@ -0,0 +1,23 @@
+from mindsdb.integrations.libs.const import HANDLER_TYPE
+from mindsdb.utilities.log import get_log
+
+logger = get_log()
+
+from .__about__ import __version__ as version, __description__ as description
+try:
+    from .cohere_handler import CohereHandler as Handler
+    import_error = None
+    logger.info("No error Importing Cohere ML Handler")
```

Comment:
Probably best to phrase this as `"Cohere ML engine successfully imported"`.

Reply:
Resolved the comment, thanks for reviewing